packages/jest-circus/src/index.ts

```diff
@@ -1,206 +1,4 @@
-  }
-
-  dispatch({blockName, mode, name: 'finish_describe_definition'});
-};
-
-const _addHook = (
-  fn: HookFn,
-  hookType: HookType,
-  hookFn: THook,
-  timeout?: number,
-) => {
-  const asyncError = new ErrorWithStack(undefined, hookFn);
-
-  if (typeof fn !== 'function') {
-    asyncError.message =
-      'Invalid first argument. It must be a callback function.';
-
-    throw asyncError;
-  }
-
-  dispatch({asyncError, fn, hookType, name: 'add_hook', timeout});
-};
-
-// Hooks have to pass themselves to the HOF in order for us to trim stack traces.
-const beforeEach: THook = (fn, timeout) =>
-  _addHook(fn, 'beforeEach', beforeEach, timeout);
-const beforeAll: THook = (fn, timeout) =>
-  _addHook(fn, 'beforeAll', beforeAll, timeout);
-const afterEach: THook = (fn, timeout) =>
-  _addHook(fn, 'afterEach', afterEach, timeout);
-const afterAll: THook = (fn, timeout) =>
-  _addHook(fn, 'afterAll', afterAll, timeout);
-
-const test: Global.It = (() => {
-  const test = (testName: TestName, fn: TestFn, timeout?: number): void =>
-    _addTest(testName, undefined, fn, test, timeout);
-  const skip = (testName: TestName, fn?: TestFn, timeout?: number): void =>
-    _addTest(testName, 'skip', fn, skip, timeout);
-  const only = (testName: TestName, fn: TestFn, timeout?: number): void =>
-    _addTest(testName, 'only', fn, test.only, timeout);
-
-  test.todo = (testName: TestName, ...rest: Array<any>): void => {
-    if (rest.length > 0 || typeof testName !== 'string') {
-      throw new ErrorWithStack(
-        'Todo must be called with only a description.',
-        test.todo,
-      );
-    }
-    return _addTest(testName, 'todo', () => {}, test.todo);
-  };
-
-  const _addTest = (
-    testName: TestName,
-    mode: TestMode,
-    fn: TestFn | undefined,
-    testFn: (testName: TestName, fn: TestFn, timeout?: number) => void,
-    timeout?: number,
-  ) => {
-    const asyncError = new ErrorWithStack(undefined, testFn);
-
-    if (typeof testName !== 'string') {
-      asyncError.message = `Invalid first argument, ${testName}. It must be a string.`;
-
-      throw asyncError;
-    }
-    if (fn === undefined) {
-      asyncError.message =
-        'Missing second argument. It must be a callback function. Perhaps you want to use `test.todo` for a test placeholder.';
-
-      throw asyncError;
-    }
-    if (typeof fn !== 'function') {
-      asyncError.message = `Invalid second argument, ${fn}. It must be a callback function.`;
-
-      throw asyncError;
-    }
-
-    return dispatch({
-      asyncError,
-      fn,
-      mode,
-      name: 'add_test',
-      testName,
-      timeout,
-    });
-  };
-
-  test.each = bindEach(test);
-  only.each = bindEach(only);
-  skip.each = bindEach(skip);
-
-  test.only = only;
-  test.skip = skip;
-
-  return test;
-})();
-
-const it: Global.It = test;
-
-export = {
-  afterAll,
-  afterEach,
-  beforeAll,
-  beforeEach,
-  describe,
-  it,
-  test,
-};
+export {addEventHandler, getState} from './state';
```

Comment:
changing the export of `jest-circus` is a breaking change.

It was also on purpose that the globals were exported at the top level, so that people could do `const {describe, it} = require('jest-circus');` whenever we support importing jest (or if e.g. cypress wants to use circus instead of mocha at some point)

Reply:
Agreed. We should be good to re-export them here to avoid breaking

Reply:
Didn't realize people were externally importing them. Thanks for calling that out.

I'll make sure they are re-exported as default (which they were before) to avoid breaking changes.

Reply:
Done and tested to be sure it continues working as before.

Reply:
great!